### DISCUSSION TITLE
When will I see the final result?

### DISCUSSION THREAD

**Discussion by aagainbeyond2** [snap-stanford/ogb#226] [PCQM4M-LSC]
When will I see the final result? Thanks

**Comment by weihua916** [comment]
The public winner announcement will be shortly after 15th. See the timeline: https://ogb.stanford.edu/kddcup2021/timeline/

**Reply by herrkun** [reply]
and when can i received email as top 3 winner of each dataset? Thanks

**Reply by weihua916** [reply]
The private email should be sent in the next few days.

**Reply by mearcstapa-gqz** [reply]
Hi, will non-top-3 participants receive email notification too when top 3 winners are privately notified? Thanks!

**Reply by weihua916** [reply]
We will only notify the winners. We will send out the notification by June 10, PT. The timeline has been made clear on the [website](https://ogb.stanford.edu/kddcup2021/timeline/).

**Reply by ZeynepP** [reply]
Are you going to publish overall ranking like the one for test submission? 
Thanks

**Reply by weihua916** [reply]
Yes, both the winners and the final leaderboards will be published.
